lib/v2/screens/profile_screens/profile/components/profile_header.dart

```diff
@@ -23,11 +26,22 @@ class ProfileHeader extends StatelessWidget {
                 children: [
                   Column(
                     children: [
-                      ProfileAvatar(
-                        size: 100,
-                        image: state.profile.image,
-                        nickname: state.profile.nickname,
-                        account: state.profile.account,
+                      InkWell(
+                        onTap: () async {
+                          final file = await showModalBottomSheet(
+                            context: context,
+                            builder: (context) => const EditProfilePicBottomSheet(),
+                          );
+                          if (file != null) {
```

Comment:
Question: what if file is null? Do we need to do something ?

Reply:
Well, it does nothing, it only looks back at the profile screen.

The return defualt value is null in the navigation, only a file is returned if the user selects an image from their gallery or takes a photo.